src/server/lib/api.js

```diff
@@ -1,8 +1,11 @@
 function foo1(a, b, a) {
     typeof a === undefined
     console.log('value of the second a:', a);
+    console.log("value of the second a:", a);
+    console.log('"value of the second a:"', a);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](http://sh.se.codacy.org/gh/codacy-acme/nodeproject/pullRequest?prid=17)
```suggestion
    console.log("\"value of the second a:\"", a);
```